### DISCUSSION TITLE
Data Saving with cognition.run - output (.csv-file) format

### DISCUSSION THREAD

**Discussion by fin329** [jspsych/jsPsych#2716] [Q&A] [answered]
Hello,
I'm running my experiment on cognition.run and it's working perfectly fine, till it comes to the data output. The preview looks fine, but if I'm downloading the csv the data is packed in the first box and parts of it randomly spread over the row. Its possible to convert the data by editing it in Excel, but since its displayed in order before, I'm wondering if its possible to download the data previewed. Did I miss anything in the settings for data download or is there anyone else with a similar problem?

Thanks in advance!

**Comment by jodeleeuw** [comment]
Tagging @javidalpe on this one

**Comment by javidalpe** [comment, marked as answer]
Hello @fin329. This seems to be an uncommon issue. Would you mind sending me more details using this form? Thanks! https://docs.google.com/forms/d/e/1FAIpQLSdYg3h6ESzd81rHlGlAib_kXA56ERuy0MBM1CwPeUdTv4lvcQ/viewform?usp=sf_link

**Reply by fin329** [reply]
Hey @jodeleeuw thanks for tagging and thanks for reaching out @javidalpe !
I filled in the form on this issue. If someone else is struggling with similar issues opening the file with different software (LibreOfficeCalc or Numbers) worked for me. That might be easier than merging and rearranging the data manually.
